Source/Scene/Globe.js

```diff
@@ -430,7 +430,16 @@ define([
         var cartesian = ellipsoid.cartographicToCartesian(cartographic, scratchGetHeightCartesian);
 
         var ray = scratchGetHeightRay;
-        Cartesian3.normalize(cartesian, ray.direction);
+        var surfaceNormal = ellipsoid.geodeticSurfaceNormal(cartesian, ray.direction);
+
+        // compute origin point, to account for a case where the terrain is under ellipsoid surface
+        var minimumHeight = Math.min(defaultValue(tile.data.minimumHeight, 0.0), 0.0);
```

Comment:
Do you think it would be safer to use lower value instead of `0.0` in `defaultValue(tile.data.minimumHeight, 0.0)`?
for example `-11500.0` (taken from [here](https://github.com/duvifn/cesium/blob/fix_billboardOnTerrain/Source/Scene/QuadtreePrimitive.js#L648))